pandas/core/reshape/merge.py

```diff
@@ -743,6 +745,46 @@ def get_result(self) -> DataFrame:
 
         return result.__finalize__(self, method="merge")
 
+    def _anti_join_update(self):
+        """
+        Converts `anti_left`, `anti_right` and `anti_full` configurations into `left`,
+        `right` and `outer` join configurations.
+        Calls `_anti_helper` with the indices or columns to be merged on.
+        """
+        if self.left_index and self.right_index:
+            # Merge using `right_index` and `left_index`
+            join_index_l, join_index_r, self.how = _anti_helper(
+                self.left.index, self.right.index, self.how
+            )
+        elif self.on is not None or (
+            None not in self.left_on and None not in self.right_on
+        ):
+            # Merge using `on` or `left_on` and `right_on`
```

Comment:
to be honest these comments are obvious, as you said. it was more the general concept that you could have expanded in the main docstring, why you have implemented it this way and what is the advantage. Just in case you come back to look at it after two years, or another developer comes in and tries to figure out what you are doing.

Reply:
the comments in merge.py looks sufficient to me. But again, I wrote it and a second pair of eyes can give a fresh perspective.
Do you think the logic in merge.py is too difficult to follow? (if you could point to any specific portion)